lms/djangoapps/edxnotes/helpers.py

```diff
@@ -141,15 +137,13 @@ def preprocess_collection(user, course, collection):
     store = modulestore()
     filtered_collection = list()
     cache = {}
+    include_path_info = ('course_structure' not in settings.NOTES_DISABLED_TABS)
```

Comment:
Ah, yes, which is why having a `NOTES_ENABLED_TABS` setting instead would have been simpler to read.